hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SstFilteringService.java

```diff
@@ -124,6 +116,37 @@ public void start() {
 
   private class SstFilteringTask implements BackgroundTask {
 
+
+    /**
+     * Marks the SSTFiltered flag corresponding to the snapshot.
+     * @param snapshotInfoTable reference to snapshotInfo table
+     * @param volume Volume name of the snapshot
+     * @param bucket Bucket name of the snapshot
+     * @param snapshotName Snapshot name
+     * @throws IOException
+     */
+    private void markSSTFilteredFlagForSnapshot(
+        Table<String, SnapshotInfo> snapshotInfoTable, String volume,
+        String bucket, String snapshotName) throws IOException {
+      boolean acquiredSnapshotLock = ozoneManager.getMetadataManager().getLock()
+              .acquireWriteLock(SNAPSHOT_LOCK,
+                  volume, bucket, snapshotName);
+      try {
+        // mark the snapshot as filtered by writing to the file
+        String snapshotTableKey = SnapshotInfo.getTableKey(volume, bucket,
+            snapshotName);
+        SnapshotInfo snapshotInfo = snapshotInfoTable.get(snapshotTableKey);
+
+        snapshotInfo.setSstFiltered(true);
+        snapshotInfoTable.put(snapshotTableKey, snapshotInfo);
+      } finally {
+        if (acquiredSnapshotLock) {
```

Comment:
wouldn't this always be true ?

Reply:
yeah it would be always true, but wrote the code as per the expected contract of the function.